### ISSUE TITLE
option to link outputs volume

### ISSUE THREAD

**Issue by mfauvain** [owntone/owntone-server#1543]
Would it be possible to link multi-outputs volume so that when an output volume changes, the rest of the outputs follow? The idea, if that makes sense, is to be able to control a multiroom volume with the physical buttons of the outputs (a remote control, directly on the airplay2 speaker, etc)
One can write a small daemon that tracks changes and act upon it, but I thought that would be a good idea to have it embedded.
Thx

**Comment by ejurgensen** [comment]
Specifically what volume control interface are you thinking of? Is it buttons on AP2 speakers? Most remotes change the master volume, thus changing all volumes like you suggest, so I don't think any change is required there.

**Comment by mfauvain** [comment]
I have some b&o speakers (airplay2) with volume controls that reflects in owntone, appletv with remote (that for some reason doe not reflect in owntone.. I thought it did) and some airplay1 airport express. 
For example I'd like to control all volumes (or main) via a b&o speaker volume control (touch control)

**Comment by ejurgensen** [comment]
I don't remember volume control for atv, and I suppose the express doesn't have any. But for the b&o it might be possible to add an Owntone setting declaring that it changes volume globally - is that what you are after?

**Comment by mfauvain** [comment]
Yeah that would be great. And to clarify, the atv volume is controled by the apple remote, however changing the volume with the atv remote is not reflected in owntone, is this expected behaviour or am I missing a setting?

**Comment by ejurgensen** [comment]
I don't remember how atv volume control works (technically), so can't say what should be expected. I suspect it might be tied into Apple Homekit.

**Comment by mfauvain** [comment]
ok, I would have thought ATV to be airplay2 as well (ie same protocol for sound)

**Comment by aleszczynskig** [comment]
Hello, don't your stereo paired HomePods update volume levels automatically? Mine seem too, at least this is my anecdotal observation. > you could monitor ```https://owntone.github.io/owntone-server/json-api/#get-an-output``` and set the main volume based on the change? shouldn't be too hard using node-red (I tend to do the same for my 'stereo zone' homepods)

**Comment by mfauvain** [comment]
Yep, the idea though was to have this daemon embedded to owntone  so it would be easy to 'link/unlink' speakers
